pandas/io/json/_json.py

```diff
@@ -832,13 +845,30 @@ def __init__(
             self.chunksize = validate_integer("chunksize", self.chunksize, 1)
             if not self.lines:
                 raise ValueError("chunksize can only be passed if lines=True")
+            if self.engine == "pyarrow":
+                raise ValueError(
+                    "currently pyarrow engine doesn't support chunksize parameter"
```

Comment:
Could you add tests with `pytest.raises(ValueError, match="...")` testing this exception (and the new ones below)?